laserscan_multi_merger:
  ros__parameters:
    destination_frame: "base_link"
    cloud_destination_topic: "/merged_cloud"
    scan_destination_topic: "/scan"
    laserscan_topics: "/lidarkanan/scan /lidarkiri/scan"
    angle_min: -3.14
    angle_max: 3.14
    angle_increment: 0.013089969754219055
    scan_time: 0.02500000037252903
    range_min: 0.023000000044703484
    range_max: 60.0


